go.mod

```diff
@@ -40,71 +38,6 @@ require (
 	gotest.tools v2.2.0+incompatible
 )
 
-require (
-	cloud.google.com/go v0.94.1 // indirect
```

Comment:
is it normal all of the below dependencies are removed?

Reply:
hmmm somehow that showed up in my `go mod tidy`.